packages/kbn-user-profile-components/package.json

```diff
@@ -2,8 +2,5 @@
   "name": "@kbn/user-profile-components",
   "version": "1.0.0",
   "private": true,
-  "main": "./target_node/index.js",
-  "browser": "./target_web/index.js",
-  "types": "./target_types/index.d.ts",
   "license": "SSPL-1.0 OR Elastic License 2.0"
-}
+}
```

Comment:
Did you mean to remove the trailing linebreak?

Reply:
That is how our .editorconfig file is setup https://github.com/elastic/kibana/blob/main/.editorconfig#L12-L13